sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Content/GetManifestResult.cs

```diff
@@ -31,17 +29,8 @@ internal DownloadManifestResult(string digest, ManifestMediaType mediaType, Bina
         public ManifestMediaType MediaType { get; }
 
         /// <summary>
-        /// The serialized content that was downloaded.
+        /// The serialized manifest.
```

Comment:
it's not serialized, is it?

Reply:
The BinaryData property holds the raw JSON and not a .NET model type.  Do you not call this serialized?

Reply:
Happy to change, though - I'll go look at what Java has and copy.